src/features/testing/workflows/convert.md

```diff
@@ -1,74 +1,51 @@
-# Convert Workflows
+# Convert Type Workflows
 
-> Before you dive in, read up on the [workflow](/concepts/workflows.md) concept first.
+_For general documentation on utilizing the Workflows tab - click [here](../workflows.md)._
 
-Convert Workflows allow you to create actions using strings as input.
+---
 
-These workflows are triggered manually and are integrated in various areas of Caido:
+While Passive and Active Workflows operate on requests and responses - `Convert Workflows` operate on **bytes** received as input.
 
-<img alt="Convert context menu" src="/_images/workflow_convert_context_menu.png"/>
+## Using Convert Workflows
 
-## Special Nodes
+---
 
-Most convert nodes have simple inputs that are covered by the [workflow](/concepts/workflows.md) guide.
+Convert Workflows are **manually triggered** and are integrated in various areas of Caido:
 
-There are a few exceptions that require further explanation:
+**When using [Constant Values](/concepts/nodes.md#1-constant-value-type)**:
 
-- [JavaScript node](#javascript)
-- [Shell node](#shell)
-- [Control flow nodes](#control-flow)
+- Within the **Workflow Editor** supply the input in the `Data` field under `Inputs` with the `Use reference` checkbox **_deselected_**. Then apply the conversion by clicking the `Save and Run` button.
 
-### JavaScript
+**When using [Reference Values](/concepts/nodes.md#2-reference-value-type)**:
 
-`Javascript` code nodes allow you to run custom scripts in your convert workflow.
-They have a minimal code editor available in the properties.
+- At the bottom of the Concept **Workflow Editor** is an Input pane in which you can supply data and then apply the conversion by clicking the `Save and Run` button.
+- Highlight the data to be changed, right-click in the associated pane and select either `Convert (Preview)` or `Convert (Replace)`. _**Preview** will present the output in a pop-up window and **Replace** will apply the conversion to the selected bytes._
```

Comment:
That section was unclear to me since it doesn't really relate to the image under it. I am not sure we need to re-explain here what the difference is between Constant and Reference, maybe more on how to use it.

Reply:
Should I just include another picture of inputting the value in the Node?

Reply:
Most convert are used with the right-click menu for preview/replace. We can show the panel we have in the editor at the bottom but it is really mostly for testing.